packages/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -71,8 +69,7 @@ export class TinyliciousClient {
 			config: {},
 		});
 
-		// eslint-disable-next-line import/no-deprecated
-		const rootDataObject = await requestFluidObject<IRootDataObject>(container, "/");
+		const rootDataObject = (await container.getEntryPoint()) as IRootDataObject;
```

Comment:
can getEntryPoint ever return undefined in this case ? If so, do we need to assert ?

Reply:
I don't think so. TinyliciousClient imports the loader package itself, thus it has the newer version with the `entryPoint` code.

Reply:
Worth asserting though? Totally up to you, I'm not digging into the details just asking.